src/app/components/table/Table.jsx

```diff
@@ -154,39 +157,51 @@ class Table extends PureComponent {
       f.map("values")
     );
 
+    const isTaxonomy = table && isTaxonomyTable(table);
+    const isSettings = table && table.type === "settings";
+    const tableClass = buildClassName("tableaux-table", {
+      taxonomy: isTaxonomy,
+      settings: isSettings,
+      generic: !isTaxonomy && !isSettings
+    });
+
     return (
       <section
         id="table-wrapper"
         tabIndex="-1"
         onMouseDown={this.onMouseDownHandler}
       >
-        <div className="tableaux-table">
-          <VirtualTable
-            key={`virtual-table-${f.get("id", table)}`}
-            actions={actions}
-            columns={columns}
-            visibleColumns={visibleColumns}
-            ref={this.findAndStoreTableDiv}
-            rows={rows}
-            tableView={tableView}
-            displayValues={displayValues}
-            table={table}
-            tables={tables}
-            langtag={langtag}
-            selectedCell={(tableView && tableView.selectedCell) || {}}
-            selectedCellEditing={(tableView && tableView.editiong) || false}
-            expandedRowIds={tableView.expandedRowIds}
-            fullyLoaded={this.props.fullyLoaded}
-            openCellContextMenu={this.showRowContextMenu}
-            closeCellContextMenu={this.hideRowContextMenu}
-            navigate={navigate}
-            finishedLoading={finishedLoading}
-            selectedCellExpandedRow={this.state.selectedCellExpandedRow}
-            setSelectedCellExpandedRow={this.setSelectedCellExpandedRow}
-            getSelectedCellExpandedRow={this.getSelectedCellExpandedRow}
-            visibleColumnOrdering={visibleColumnOrdering}
-            hasStatusColumn={hasStatusColumn}
-          />
+        <div className={tableClass}>
+          {isTaxonomyTable(table) ? (
```

Comment:
[**Minor**] Die `isTaxonomyTable` Funktion wurde ja oben schon aufgerufen und das Ergebnis ist in `isTaxonomy` gespeichert. Könnte man auch direkt das abfragen, als nochmal die Funktion aufzurufen.

```suggestion
          {isTaxonomy ? (
```